src/D2L.Bmx/Okta/State/OktaAccountState.cs

```diff
@@ -0,0 +1,10 @@
+using D2L.Bmx.Okta.Models;
+namespace D2L.Bmx.Okta.State;
+
+internal record OktaAccountState( OktaApp[] OktaApps, string AccountType ) {
+
+	public string[] Accounts
+		=> this.OktaApps.Where( app => app.AppName == AccountType ).Select( app => app.Label ).ToArray();
```

Comment:
I think this could be hardcoded to "amazon_aws" but again, something to do when refactoring further once its all working.